test/jdk/java/net/vthread/HttpALot.java

```diff
@@ -23,13 +23,18 @@
 
 /**
  * @test
+ * @bug 8294610
  * @summary Stress test the HTTP protocol handler and HTTP server
  * @requires vm.debug != true
  * @modules jdk.httpserver
  * @library /test/lib
  * @compile --enable-preview -source ${jdk.version} HttpALot.java
+ * @comment This test runs with -Dsun.net.httpserver.nodelay=true
+ *          to enable TCP_NODELAY on the sockets "accept()"ed by the HttpServer. This is
+ *          to avoid occasional 40ms delays receiving responses from the server on Linux.
```

Comment:
Thanks for the update the bug tag and improving the comment. Just a minor nit is that you might want to re-flow it to avoid 1 short line and two longer lines. Ignore if you want.

Reply:
Hello Alan, what you suggest seems fine. So I've now updated the comment accordingly.